pkg/controller/jobs/job/job_controller.go

```diff
@@ -311,9 +312,11 @@ func (j *Job) podsCount() int32 {
 }
 
 func (j *Job) minPodsCount() *int32 {
-	if strVal, found := j.GetAnnotations()[JobMinParallelismAnnotation]; found {
-		if iVal, err := strconv.Atoi(strVal); err == nil {
-			return pointer.Int32(int32(iVal))
+	if features.Enabled(features.PartialAdmission) {
```

Comment:
this should be checked in the framework code, not the job implementation.

Reply:
to do it in the framework we'll need to get the podsets and the set all mins to nil if the featuregate is not enabled ... in a couple of places

Reply:
I think that's the correct way.

Reply:
...  done